index.js

```diff
@@ -0,0 +1,5 @@
+const path = require('path')
+
+module.exports = {
+  folderPath: path.resolve(__dirname, 'themes')
```

Comment:
Please rename this to `themesDirectory`.

```suggestion
	themesDirectory: path.resolve(__dirname, 'themes')
```